dev/Expander/Expander.idl

```diff
@@ -1,6 +1,15 @@
 ﻿namespace MU_XC_NAMESPACE
 {
 
+[WUXC_VERSION_MUXONLY]
```

Comment:
>WUXC_VERSION_MUXONLY [](start = 1, length = 20)

These template settings need to be code reviewed... @ranjeshj and @mikehillberg

Reply:
got it